README.md

```diff
@@ -1,5 +1,4 @@
-
-# reckoner [![CircleCI](https://circleci.com/gh/reactiveops/reckoner.svg?style=svg)](https://circleci.com/gh/reactiveops/reckoner)
+# reckoner [![CircleCI](https://circleci.com/gh/reactiveops/reckoner.svg?style=svg)](https://circleci.com/gh/reactiveops/reckoner) [![codecov](https://codecov.io/gh/reactiveops/reckoner/branch/master/graph/badge.svg)](https://codecov.io/gh/reactiveops/reckoner)
```

Comment:
This might be a holdover from the other PR? Not sure if it's supposed to be in here

Reply:
I think maybe I forgot it, or maybe it is a holdover.  Either way I'll take care of it when I rebase.

Reply:
I must have forgotten to add the codecov badge in the other PR.  I added it to the description on this one as well as removing the duplicate call to codecov.